instance/tests/test_tasks.py

```diff
@@ -146,6 +170,22 @@ def test_make_appserver_active(self, active, mock_make_active):
         tasks.make_appserver_active(self.appserver.id, active=active)
         mock_make_active.assert_called_once_with(active=active)
 
+    @ddt.data([False, True], [False, False])
+    @patch('instance.models.server.OpenStackServer.update_status')
+    @patch('instance.models.openedx_appserver.OpenEdXAppServer.make_active')
+    # pylint: disable=invalid-name
```

Comment:
Ah snap, I didn't know this would introduce a pylint violation :) In that case, sorry for my old comment -- it's probably better to leave this as you originally had it.